src/lib.rs

```diff
@@ -42,9 +42,18 @@
 //!
 
 #![no_std]
-
+#![cfg(not(feature = "std"))]
 use core::ops::{Bound, RangeBounds};
 
+#[cfg(feature = "std")]
+use std::ops::{Bound, RangeBounds};
```

Comment:
```suggestion
use core::ops::{Bound, RangeBounds};
```

You should be able to use `core::*` even with `std`

Reply:
The rust docs say that it's better to use std instead of core, but it doesn't elaborate on that

Reply:
As a compromise you should be able to do something like this to avoid duplicate imports:

```
#[cfg(not(feature = "std"))
use core as std;

use std::whatever;
```